doc/relevancy.md

```diff
@@ -458,6 +459,26 @@ but something else still gets high traffic ([03c4bfa][], May 2015).
 If we had other signals of content quality we might be able to reduce our
 dependence on the page views.
 
+### Finding underperforming queries
+
+1. Go to the [datastudio report][]
```

Comment:
The report does not auto-update so will be for the dates displayed, but you can rerun the BigQuery queries to get new data